### DISCUSSION TITLE
is it possible to read two pages as a single page and can extract the table

### DISCUSSION THREAD

**Discussion by kathimohan** [jsvine/pdfplumber#781] [Q&A]
The table is extended for more than a single page, and appending the table data later not working for all, is it possible to read two pages as a single page and can extract the table or is there any way to remove pagebreaks.

**Comment by jsvine** [comment]
At the moment, there is no standard way of doing this in pdfplumber. I haven't come up with an approach that would be generally applicable to the situation. But if you or anyone else devise a method, I would happily review an implementation proposal or pull request.

**Reply by yonglee7015** [reply]
for my situation, first loop all pages if there is a table, then store the page nubmer(page with table) in a pageNo list.   
then  split pageNo list  based on if it is a continuous page.   for example[1,2,4,9,10] will be split to [[1,2],[4],[9,10]] . Then based on
this new list , loop and append table.  Anyway there is a bug, if continuous page is not a continuous table.

**Reply by kathimohan** [reply]
appending tables page-wise is not producing a correct result, for example the same row of a table is split between two pages, the first page provides it as one row, and the second page's same row as the next row.So, need to combine both the pages as one and extract that table, is there any solution for this

**Reply by yonglee7015** [reply]
maybe you can write some code to ignore the first line for next page. some code like this table(1:).

**Reply by chanpreet90** [reply]
is it possible for you to share the PDF?

**Reply by kathimohan** [reply]
> is it possible for you to share the PDF?

I can't share pdf, but problem solved by counting the horizontal lines
